src/utils/npm-utils.js

```diff
@@ -3,31 +3,66 @@
 const path = require('path');
 const spawn = require('cross-spawn');
 
-module.exports = {
-  installDependency: (options, callback) => {
-    const { dependency, installPath, isDev, save } = options;
+const buildInstallCommand = options => {
+  const args = ['install'];
+
+  if (options.save) {
+    args.push('--save-exact');
+    args.push(options.isDev ? '--save-dev' : '--save');
+  }
 
-    const args = ['install'];
+  return args;
+};
+
+const executeCommand = (options, callback) => {
+  const cmd = spawn('npm', options.command, {
+    cwd: options.installPath,
+    stdio: 'inherit'
+  });
+
+  cmd.on('error', () => callback('error'));
+  cmd.on('close', code => callback(code !== 0 ? code : null));
+};
 
-    if (save) {
-      args.push('--save-exact');
-      args.push(isDev ? '--save-dev' : '--save');
-    }
+const moduleName = dependency => dependency.split('@')[0];
 
-    const cmd = spawn('npm', [...args, dependency], {
-      cwd: installPath,
-      stdio: 'inherit'
-    });
+module.exports = {
+  installDependencies: (options, callback) => {
+    const { dependencies, installPath } = options;
+    const npmi = buildInstallCommand(options);
+    const cmdOptions = { installPath, command: [...npmi, ...dependencies] };
+
+    executeCommand(cmdOptions, err =>
+      callback(
+        err,
+        err
+          ? null
+          : {
+            dest: dependencies.map(dependency =>
+                path.join(installPath, 'node_modules', moduleName(dependency))
+              )
+          }
+      )
+    );
+  },
+  installDependency: (options, callback) => {
+    const { dependency, installPath } = options;
+    const npmi = buildInstallCommand(options);
+    const cmdOptions = { installPath, command: [...npmi, dependency] };
 
-    cmd.on('error', () => callback('error'));
-    cmd.on('close', code => {
-      const err = code !== 0 ? code : null;
+    executeCommand(cmdOptions, err =>
```

Comment:
That seems like a lot of duplication just to handle a single vs multiple deps install, coudln't we make into a single method?